test/unit/test_host.rb

```diff
@@ -65,8 +65,14 @@ def test_assert_hosts_hash_equally
     end
 
     def test_assert_hosts_compare_equal
-      assert Host.new('example.com').eql? Host.new('example.com')
-      assert Host.new('example.com').equal? Host.new('example.com')
+      h1 = Host.new('first_host')
+      h2 = Host.new('second_host')
+      h1.expects(:hash).times(3).returns('example.com')
```

Comment:
Hey - yes I did see your note on 275 - sorry I didn't know where it was best to reply.

I understand what you mean about testing just one thing at a time. 

But I don't think it helps to set up things which are purposefully different to what happens in the real app. I think this would be very confusing to someone reading the test who didn't realise you had done this. This test reads like different hosts should return the same string hash which is the domain string, and they should be equal.

So if it's OK with you and rubocop, I would be in favour of removing the hash stubbing lines in this case.

As a side note, I think the implementation of the equals methods probably shouldn't use the hash method anyway since there is a (very slim) chance of collisions, but obviously this is unrelated to the rubocop change.